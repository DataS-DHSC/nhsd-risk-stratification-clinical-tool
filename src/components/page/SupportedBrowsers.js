import React from 'react';
import PageReviewDates from '../PageReviewDates';
import SiteBrandingTitle from '../SiteBrandingTitle';

const SupportedBrowsers = () => (
  <>
    <div className="nhsuk-grid-row">
      <div className="nhsuk-grid-column-two-thirds">
        <h1 id="supported-browsers-heading">
          Browsers Supported by the <SiteBrandingTitle isDefaultName />
        </h1>
      </div>
    </div>
    <div className="nhsuk-grid-row">
      <div className="nhsuk-grid-column-two-thirds">
        <table id="supported-browsers-table">
          <tr>
            <th>Operating System</th>
            <th>Browser</th>
          </tr>
          <tr>
            <td>Windows</td>
            <td>Internet Explorer 11</td>
          </tr>
          <tr>
            <td />
            <td>Edge 84 (latest)</td>
          </tr>
          <tr>
            <td />
            <td>Chrome 86 (latest)</td>
          </tr>
          <tr>
            <td />
            <td>Firefox 82 (latest)</td>
          </tr>
          <tr>
            <td>Mac OS</td>
            <td>Safari 14.0 (latest)</td>
          </tr>
          <tr>
            <td />
            <td>Chrome 86 (latest)</td>
          </tr>
          <tr>
            <td>iOS</td>
            <td>Safari 14.0 (latest)</td>
          </tr>
        </table>
      </div>
    </div>
    <PageReviewDates page="static" />
  </>
);

export default SupportedBrowsers;
