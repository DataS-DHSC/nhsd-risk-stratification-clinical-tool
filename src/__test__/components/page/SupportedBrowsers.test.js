import React from 'react';
import { render } from '@testing-library/react';
import SupportedBrowsers from '../../../components/page/SupportedBrowsers';

const setup = () => render(<SupportedBrowsers />);

describe('SupportedBrowsers', () => {
  it('should match snapshot', () => {
    const container = setup();

    expect(container.asFragment()).toMatchSnapshot();
  });
});
