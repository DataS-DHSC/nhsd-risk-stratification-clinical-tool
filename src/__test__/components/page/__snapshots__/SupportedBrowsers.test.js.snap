// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportedBrowsers should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <h1
        id="supported-browsers-heading"
      >
        Browsers Supported by the COVID-19 Clinical Risk Assessment Tool
      </h1>
    </div>
  </div>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <table
        id="supported-browsers-table"
      >
        <tbody>
          <tr>
            <th>
              Operating System
            </th>
            <th>
              Browser
            </th>
          </tr>
          <tr>
            <td>
              Windows
            </td>
            <td>
              Internet Explorer 11
            </td>
          </tr>
          <tr>
            <td />
            <td>
              Edge 84 (latest)
            </td>
          </tr>
          <tr>
            <td />
            <td>
              Chrome 86 (latest)
            </td>
          </tr>
          <tr>
            <td />
            <td>
              Firefox 82 (latest)
            </td>
          </tr>
          <tr>
            <td>
              Mac OS
            </td>
            <td>
              Safari 14.0 (latest)
            </td>
          </tr>
          <tr>
            <td />
            <td>
              Chrome 86 (latest)
            </td>
          </tr>
          <tr>
            <td>
              iOS
            </td>
            <td>
              Safari 14.0 (latest)
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div
    class="nhsuk-review-date"
    id="review-date"
  >
    <p
      class="nhsuk-body-s"
    >
      Page last reviewed: 1 December 2020
      <br />
      Next review due: 23 December 2023
    </p>
  </div>
</DocumentFragment>
`;
